src/views/Give/RedeemYieldModal.tsx

```diff
@@ -26,10 +26,12 @@ type RedeemModalProps = {
   isModalOpen: boolean;
   callbackFunc: RedeemSubmitCallback;
   cancelFunc: RedeemCancelCallback;
-  deposit: BigNumber;
-  redeemableBalance: BigNumber;
+  deposit: DecimalBigNumber;
+  redeemableBalance: DecimalBigNumber;
 };
 
+const DECIMAL_PLACES = 2;
```

Comment:
same decimal question here.

Reply:
<img width="765" alt="image" src="https://user-images.githubusercontent.com/80423742/162446910-f4def7a2-121d-4e3b-a670-92792bf5a4b2.png">
<img width="773" alt="image" src="https://user-images.githubusercontent.com/80423742/162446941-352f86b3-c04e-4f4e-941f-91777bb9560f.png">

currently it shows 4 different decimal precision levels. It's an interesting nuance we should standardize I think.